api.swagger.yaml

```diff
@@ -467,96 +576,803 @@ paths:
+              ^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$
+      responses:
+        '200':
+          description: Returns a single complete group object.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/group-schema'
+        '400':
+          description: The supplied UUID was malformed.
+        '404':
+          description: The UUID does not correspond to an existing group.
+      description: Get information about the group.
+  '/groups/{group_id}/members':
+    get:
+      tags:
+        - groups
+      parameters:
+        - description: The Invotra UUID of the group.
+          in: path
+          name: group_id
+          required: true
+          schema:
+            type: string
+            pattern: >-
+              ^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$
+      responses:
+        '200':
+          description: |
+            List of group members. If there are no members, an empty array.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/group-membership-list-schema'
+        '400':
+          description: The supplied UUID was malformed.
+        '404':
+          description: The UUID does not correspond to an existing group.
+      description: Get a list of group members.
+  '/groups/{group_id}/member':  
+    get:
+      tags:
+        - groups
+      parameters:
+        - description: The Invotra UUID of the group to check following status against.
+          in: path
+          name: user_id
+          required: true
+          schema:
+            type: string
+            pattern: >-
+              ^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$
+      responses:
+        '200':
+          description: |
+            Membership status returned
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/common-status-schema'
+        '400':
+          description: The UUID was malformed.
+        '404':
+          description: The group does not exist.
+      description: Check if the current user is a member of a group.
+  '/groups/{group_id}/members/{user_id}':
+    delete:
+      tags:
+        - groups
+      parameters:
+        - description: The Invotra UUID of the group.
+          in: path
+          name: group_id
+          required: true
+          schema:
+            type: string
+            pattern: >-
+              ^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$
+        - description: The Invotra UUID of the user.
+          in: path
+          name: user_id
+          required: true
+          schema:
+            type: string
+            pattern: >-
+              ^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$
+      responses:
```

Comment:
Looking at this one it occurred to me that it may also need a 403 response for when you're neither the user nor a group admin. Then I thought maybe we need 403s for a few others, eg private/secret groups that you're not a member of. But maybe that should be a separate PR to add all the 403s

Reply:
Will do in a separate PR